src/sentry/static/sentry/app/components/group/sentryAppExternalIssueForm.tsx

```diff
@@ -21,10 +21,12 @@ import {replaceAtArrayIndex} from 'app/utils/replaceAtArrayIndex';
 //0 is a valid choice but empty string, undefined, and null are not
 const hasValue = value => !!value || value === 0;
 
-type FieldFromSchema = Field & {
+type FieldFromSchema = Omit<Field, 'choices' | 'type'> & {
+  type: 'select' | 'textarea' | 'text';
   default?: string;
   uri?: string;
   depends_on?: string[];
+  choices?: Array<[any, string]>;
```

Comment:
Should `choices` be optional type? I think for a Select we can assume that there would always be an array of choices.

Reply:
@NisanthanNanthakumar Nah, sometimes there aren't choices passed in from the backend and we fetch it when the user starts typing.